### ISSUE TITLE
Rescan vs Rebuild is confusing

### ISSUE THREAD

**Issue by Glazed** [RupertAvery/DiffusionToolkit#52]
I just spent 45 minutes testing and writing up a bug that I didn't submit because it's not a bug. Instead, the UI is simply confusing. 

### Problem:

I was clicking "Rebuild" to try to get the app to scan the new folder I added, but apparently the Rebuild only updates _existing_ files if their metadata has changed. even though the popup that explains what this button does, I thought that was going to add new files. This mistaken idea of mine was reinforced by the popup that appears after the Rebuild which says, "N files updated" where N was the **full** count of images in all folders, _including the new ones it didn't actually add_.

### Steps:

1. Start the program without an existing .db file or config.json.
2. Run the app, and when it opens the settings window for you, add a test folder containing X images.
3. Let the app do its initial scan.
4. Note that the status bar now says "X images in database".
5. Open Settings and add a second folder containing Y images
6. Click the Rebuild button.
7. A popup will appear which says "Rebuild complete. X+Y images updated". 
8. Hit OK.
9. Note that the app still says "X images in database", despite the popup indicating that it updated X+Y images.

### Expected behavior:

In step 7 above, it should say "X images updated", not X+Y.

### Enhancement suggestions:

1. Have that popup say "X images updated. Y new images found, but not yet added. Use the Rescan Folders button to add them".
2. When adding new folders, have the app suggest a Rescan when closing the settings window.
3. Use different icons. The classic "refresh" circle of arrows used for the Rebuild button feels like it should do a scan to add new files. The magnifying glass icon (despite having a + in the middle) looks like a search icon. I didn't know what it did, but I didn't think it would scan my newly added folders. I thought it was some sort of advanced search button or something. I totally ignored it because it just didn't register to me as the button I should press to add new files whereas the Rebuild icon did!
4. Even if someone notices that there are two buttons the difference between the word "Rescan" and "Rebuild" are not clear to someone who doesn't understand the function of the buttons. The tooltips should be labeled "Scan folders for new images", and "Rebuild metadata of existing images only".
5. The explanation in the confirmation popup when you press Rebuild isn't clear enough. "This will update the metadata in the database with newly scanned metadata from the files". To me that sounds like exactly what I wanted to do. It uses just "the files" in that first sentence. To me that meant "all files in my source folders". Then, "You only need to do this if you think you're missing some metadata." Well, yes, my database is definitely missing metadata. It's missing all my new files. The first sentence should say "This will update the metadata of only the existing files in the database with current metadata from the files themselves." I think that is much more clear. The second sentence should read something like, "You only need to do this if you've updated the metadata in the files since they were added." I don't see any other reason the database would be missing metadata unless maybe some new version of the app scans for new metadata that an old version ignored. If that's a possibility, then maybe "You only need to do this if you've updated the metadata in the files since they were added or if they contain metadata that an older version of this program didn't store." Something like that. You might as well be really generous with the explanation here. It confused me, and I'm no dummy.
6. Move the Rebuild function to a menu and not the main toolbar. It's not something I see users needing very often. And then reserve the "Refresh" icon for the Rescan feature. Or not. Maybe the magnifying glass icon is good enough when there's not a refresh icon right next to it to distract users like me. :D

Thanks for the great program!

**Comment by RupertAvery** [comment]
Thanks for the feedback!

Will do #6, as it makes more sense.

I have also fixed the Rebuild Metadata result to display X images updated.

Is this text more clear for Rebuild Metadata?

This will update the metadata of ALL existing files in the database with current metadata in actual files.

You only need to do this if you've updated the metadata in the files since they were added or if they contain metadata that an older version of this program didn't store.

Are you sure you want to continue?
